### DISCUSSION TITLE
[Feature] Expose email templates

### DISCUSSION THREAD

**Discussion by mprajescu** [mediacms-io/mediacms#609] [Feature Requests]
A necessity for setting up/customizing email templates for communications from the application. 
The initial template needs to be simple, but customizabile, with own logo and possibly another text

**Comment by mgogoulos** [comment]
That's a good suggestion, and an easy addition that can have a great impact

**Comment by mgogoulos** [comment]
Create a folder with all email templates, and move existing templates there (as user and media upload related)
